gradle/idea.gradle

```diff
@@ -1,215 +0,0 @@
-apply plugin: 'java-library'
-apply plugin: 'idea'
-
-assert sourceCompatibility instanceof JavaVersion;
-assert targetCompatibility instanceof JavaVersion;
-assert sourceCompatibility == targetCompatibility;
-
-project.tasks.withType(JavaCompile) { JavaCompile task ->
-  // This actually runs javax.tools.JavaCompiler in-process, but we can still pass arguments
-  // as though it were running the javac executable.
-  task.options.compilerArgs << "-XDignore.symbol.file";
-  task.options.compilerArgs << "-Xlint:none";
-  task.options.compilerArgs << '-encoding' << 'utf-8';
-  task.options.encoding = 'UTF-8'
```

Comment:
defaulting to gradle-baseline's IntelliJ Idea integration configs. Checkstyle now works properly and compiles fine for me in IDEA 2020.3 CE EAP.